src/server/login.js

```diff
@@ -82,6 +82,7 @@ module.exports = function (client, server, options) {
         }
         // Convert to a valid UUID until the session server updates and does
         // it automatically
+        client.username = profile.name
         client.uuid = profile.id.replace(/(\w{8})(\w{4})(\w{4})(\w{4})(\w{12})/, '$1-$2-$3-$4-$5')
```

Comment:
You've placed this just between a line and a comment reffering to that line

Reply:
Good point.